### PR TITLE
Update wrapper.conf file

### PR DESCRIPTION
**Pull request by Vathsan**
## Purpose
This PR updates the wrapper.conf file with the updated configurations we have in our documentation [1]. 
On top of that, the following configurations are added to support log4j2 upgrade. Without the following configurations the existing carbon logs will get override after restart of EI windows service.
```
wrapper.java.additional.14 = -Djava.util.logging.config.file=${carbon_home}\\conf\\etc\\logging-bridge.properties
wrapper.java.additional.40 = -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
```
[1] https://docs.wso2.com/display/EI660/Running+the+Product+as+a+Windows+Service

### CODE REVIEW

File: distribution/src/conf/wrapper.conf [hunk 1]
```diff
@@ -19,22 +19,22 @@ wrapper.tmp.path = ${jna_tmpdir}
 #********************************************************************
 # Java Application main class
 wrapper.java.app.mainclass=org.wso2.carbon.bootstrap.Bootstrap
-# Log Level for console output.  (See docs for log levels)
-wrapper.console.loglevel=INFO
+# Log Level for console output. (See docs for log levels)
+wrapper.console.loglevel=DEBUG
```
Comment (Nirothipan) [inline distribution/src/conf/wrapper.conf:23]: Any reason to make the default as debug , shall we change back to INFO

Comment (Vathsan) [reply distribution/src/conf/wrapper.conf:23]: We have the loglevel as DEBUG in our minimal wrapper.conf configuration in the doc [1]. We can keep INFO by default and let the users update the loglevel accordingly. WDYT?

[1] https://docs.wso2.com/display/EI660/Running+the+Product+as+a+Windows+Service

Comment (Nirothipan) [reply distribution/src/conf/wrapper.conf:23]: We always pack the wrapper conf as INFO and we have the same in kernel https://github.com/wso2/carbon-kernel/blob/4.6.x/distribution/kernel/carbon-home/bin/yajsw/wrapper.conf#L23.  It was added to EI 660 docs due to an issue in the packed wrapper conf in the default pack. Also IMO by default we need to provide in the INFO level and the user need to change if they need additional logs.

Comment (Vathsan) [reply distribution/src/conf/wrapper.conf]: Changed the default log level as INFO in 2b893df49e4d2501040005aa434da9ef8d8d24ca

**Pull request merged**